libs/web/project/ui/src/lib/project-ui-deadline.tsx

```diff
@@ -0,0 +1,13 @@
+import { Project } from '@deanslist-platform/sdk'
+import { Group, Text } from '@mantine/core'
+import { IconCalendar } from '@tabler/icons-react'
+
+export function ProjectUiDeadline({ project }: { project: Project }) {
+  // TODO: Add deadline on project object
```

Comment:
Let's discuss how the deadline should be stored in the db...
In dates? In end date?